docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,382 @@
+
+    If you get a "Command not found" error, re-run the installer from AdoptOpenJDK
+    and be sure to configure the installer to "Set JAVA_HOME variable".
+
+Create an Android app and compile it
+====================================
+
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. First, you run the ``create``
+command. This downloads an Android app template and adds your Python code to it.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+Once this completes, we'll now have an ``android`` directory in your project.
+This directory will contain a ``Hello World`` folder, which will contain an
+Android project configured to use gradle, your application code, and a support
+package containing the Python interpreter and the `rubicon-java` library.
+
+In the process, Briefcase downloads the Android SDK. You may be prompted to
+accept Google's Android SDK license.
+
+You can then use Briefcase's ``build`` command to compile this into an Android
+APK app file. It will take quite a few minutes the first time, depending on
+your Internet connection speed, and then should take less than 30 seconds every
+successive time. If you see that Gradle is "CONFIGURING" for many, many minutes,
+this is normal; Gradle is busy performing downloads, and you should not
+interrupt it.
```

Comment:
Probably worth calling out this "it will take many many minutes" claim out `.. admonition::` .That gives the statement emphasis.